typespec/Applications.Core/applications.tsp

```diff
@@ -38,14 +39,16 @@ using OpenAPI;
 namespace Applications.Core;
 
 @doc("Radius Application resource")
-model ApplicationResource is TrackedResource<ApplicationProperties> {
+model ApplicationResource is TrackedResourceRequired<ApplicationProperties, "applications">{
```

Comment:
What does this name suggest? Does it mean that it consists of required properties of the TrackedResource? Is this going to be a naming convention for Radius?

Reply:
It means TrackedResource with properties as required